---
title: FailFastException
ms.custom: 
ms.date: 03/30/2017
ms.prod: .net-framework
ms.reviewer: 
ms.suite: 
ms.technology: dotnet-clr
ms.tgt_pltfrm: 
ms.topic: article
ms.assetid: f30de0d9-3a91-4800-9d09-3504d7e5a10f
caps.latest.revision: "8"
author: Erikre
ms.author: erikre
manager: erikre
ms.openlocfilehash: 23fe7e4bd96fd6cffdcd6eb559d36de79443be26
ms.sourcegitcommit: 4f3fef493080a43e70e951223894768d36ce430a
ms.translationtype: MT
ms.contentlocale: es-ES
ms.lasthandoff: 11/21/2017
---
# <a name="failfastexception"></a>FailFastException
Id: 103  
  
 Gravedad: error  
  
 Categoría: seguimiento  
  
## <a name="description"></a>Descripción  
 Se inició una excepción durante FailFast. El evento enumera la excepción, el nombre del proceso y el id. del mismo.  
  
## <a name="see-also"></a>Vea también  
 [Registro de eventos](../../../../../docs/framework/wcf/diagnostics/event-logging/index.md)  
 [Referencia General de eventos](../../../../../docs/framework/wcf/diagnostics/event-logging/events-general-reference.md)
